package.json

```diff
@@ -25,14 +25,14 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/edx/frontend-component-footer.git"
+    "url": "git+https://github.com/mitodl/frontend-component-footer-mitol"
   },
   "author": "edX",
```

Comment:
I guess, should it not be MIT ?

Reply:
Yes the author should have been MIT, updated